homeassistant/components/skybell/config_flow.py

```diff
@@ -35,15 +45,19 @@ async def async_step_user(
             password = user_input[CONF_PASSWORD]
 
             self._async_abort_entries_match({CONF_EMAIL: email})
-            user_id, error = await self._async_validate_input(email, password)
-            if error is None:
-                await self.async_set_unique_id(user_id)
+            result = await self._async_validate_input(email, password)
+            if result.error is None:
+                await self.async_set_unique_id(result.user_id)
                 self._abort_if_unique_id_configured()
                 return self.async_create_entry(
                     title=email,
-                    data={CONF_EMAIL: email, CONF_PASSWORD: password},
+                    data={
+                        CONF_EMAIL: email,
+                        CONF_PASSWORD: password,
+                        CONF_MAC: result.macs,
```

Comment:
If it can contain multiple, its probably better to make a new constant so we already have a whole class of bugs telling apart `entity_id` and `entity_ids` so best to avoid that pattern if we can up front